core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -91,7 +93,12 @@ private[spark] class EventLoggingListener(
   private[scheduler] val loggedEvents = new ArrayBuffer[JValue]
 
   // Visible for tests only.
-  private[scheduler] val logPath = getLogPath(logBaseDir, appId, appAttemptId, compressionCodecName)
+  private[scheduler] val logPath = getLogPath(
+    logBaseDir, appId, appAttemptId, compressionCodecName)
```

Comment:
yes, my mistake, the width is 100, it's correct. I'll change it back later, thanks